projects/sensory_interface/public/index.html

```diff
@@ -3,15 +3,60 @@
 <title> Sensory interface </title>
 
 <head>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
     <script src="app.js"></script>
 </head>
 
 <body>
-    <textarea id="textInput">1	2	3
+    <div class="container-fluid">
+        <h2>Sensory interface</h2>
+
+        <h3>Data</h3>
+        Data in tab-delimited CSV format:<br>
+        <textarea id="textInput">1	2	3
 4	5	6</textarea>
-    <button onclick="processData()">process </button>
-    <br>
-    <div id="tableContainer"></div>
+        <br>
+        <button onclick="processData()">Process</button>
```

Comment:
If I am not mistaking, this button will show twice on the page? correct? is this intentional?